plugins/nexus-repository-manager/src/__fixtures__/repository/docker/v2/janus-idp/backstage-showcase/manifests/sha-33dfe6b.json

```diff
@@ -1,41 +1,25 @@
 {
-  "schemaVersion": 2,
-  "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
-  "config": {
-    "mediaType": "application/vnd.docker.container.image.v1+json",
-    "size": 17214,
-    "digest": "sha256:b98e210da4230ea59323687f2b6678e12f00a8c82a9433f8235a02e50a06c640"
-  },
-  "layers": [
+  "schemaVersion": 1,
```

Comment:
should these fixtures files have been duplicated and modified? I'm unsure where they are actually used 😅

Reply:
These are used in [the dev app](https://github.com/rmartine-ias/janus-backstage-plugins/blob/docker-schema-1/plugins/nexus-repository-manager/dev/index.tsx#L26), and the [tests](https://github.com/rmartine-ias/janus-backstage-plugins/blob/docker-schema-1/plugins/nexus-repository-manager/src/api/nexus-repository-manager-api-client/nexus-repository-manager-api-client.test.ts#L75). I wanted a faster feedback loop for testing, so I could get the error in the dev app. This is not necessarily still needed, the type system _should_ catch any more errors. Think I should revert this part?

Reply:
thanks, that makes sense. I think let's see if the maintainers have an opinion here

Reply:
I think it makes sense to have this included as it is present in the PR. As @rmartine-ias wrote, the fast dev feedback is really nice thing to have. 🙂